css/local.css

```diff
@@ -292,6 +306,7 @@ footer p {
   -webkit-align-self: center;
   -ms-flex-item-align: center;
   align-self: center;
+  font-family: "Montserrat", sans-serif;
```

Comment:
Yet another hardcoded font? We already determined in the last PR that we don't want that.